docs/setup/start-stop.asciidoc

```diff
@@ -2,7 +2,10 @@
 == Start and stop {kib}
 
 The method for starting and stopping {kib} varies depending on how you installed
-it.
+it.  If a password protected keystore is used, the environment variable
+`KBN_KEYSTORE_PASSPHRASE_FILE` can be used to point to a file containing the password,
```

Comment:
Should this say `KBN_KEYSTORE_PASSWORD_FILE`?

Reply:
Thank you, I pushed 51122486fcc4210ea15e8562d7ecde286a8bbefe.